phoenix-android/src/main/kotlin/fr/acinq/phoenix/android/components/Buttons.kt

```diff
@@ -266,20 +268,32 @@ fun Clickable(
 }
 
 @Composable
-fun WebLink(text: String, url: String) {
+fun WebLink(
+    text: String,
+    url: String,
+    modifier: Modifier = Modifier,
+    maxLines: Int = Int.MAX_VALUE,
+    overflow: TextOverflow = TextOverflow.Clip,
+) {
     val context = LocalContext.current
     Text(
-        modifier = Modifier.clickable(
+        modifier = modifier.clickable(
             role = Role.Button,
-            onClickLabel = stringResource(id = R.string.accessibility_link)
-        ) {
-            openLink(context, url)
-        },
+            onClickLabel = stringResource(id = R.string.accessibility_link),
+            onClick = { openLink(context, url) }
+        ),
         text = text,
+        overflow = overflow,
+        maxLines = maxLines,
         style = MaterialTheme.typography.body1.copy(textDecoration = TextDecoration.Underline, color = MaterialTheme.colors.primary)
     )
 }
 
+@Composable
+fun txLink(txId: String): String {
```

Comment:
This functionality is also available in BlockchainExplorer.kt.

Reply:
Fixed in f3b5cc54